docs/docs/cloud/developer.mdx

```diff
@@ -0,0 +1,73 @@
+---
+id: developer
+title: Ory Cloud Developer Plan
+sidebar_label: Developer Plan
+---
+
+This document describes what you can expect when signing up for a Developer Plan
+in Ory Cloud.
+
+The Developer Plan includes one Ory Cloud Project for testing, research and
+personal projects.
```

Comment:
It's unclear how the plan is applied. It sounds like you only get one developer plan per account - but you can create as many (within reason) projects as you want using a developer plan.

Reply:
Oh ok, I thought you are limited to one project for the Developer plan.

Reply:
Changed to projects now.